lesson-15/main.js

```diff
@@ -0,0 +1,427 @@
+const filterForm = document.getElementById('filterForm');
+const counterAdult = document.getElementById('counterAdult');
+const counterChildren = document.getElementById('counterChildren');
+const counterRooms = document.getElementById('counterRooms');
+const btnCounterAdultMinus = document.getElementById('btnCounterAdultMinus');
+const btnCounterAdultPlus = document.getElementById('btnCounterAdultPlus');
+const btnCounterChildrenMinus = document.getElementById('btnCounterChildrenMinus');
+const btnCounterChildrenPlus = document.getElementById('btnCounterChildrenPlus');
+const btnCounterRoomsMinus = document.getElementById('btnCounterRoomsMinus');
+const btnCounterRoomsPlus = document.getElementById('btnCounterRoomsPlus');
+const childAgeQuastion = document.getElementById('childAgeQuastion');
+const guestInput = document.getElementById('guestInput');
+
+document.addEventListener('DOMContentLoaded', (() => {
+  counterChildren.value = '0';
+  counterAdult.value = '0';
+  counterRooms.value = '0';
+}));
+
+// слайдер для Home guests loves
+
+function loadSlider() {
+  $(".slick__wrapper").slick({
+    slidesToShow: 4,
+    slidesToScroll: 4,
+    infinite: true,
+    arrows: true,
+  });
+}
+
+async function getResponse() {
+const response = await fetch('https://fe-student-api.herokuapp.com/api/hotels/popular');
+const data = await response.json();
+ 
+ console.log(data)
+
+ const homesInfoPanel = document.querySelector(".favorite-wrapper");
+
+ data.forEach((item) => { const hotelsItem = document.createElement("div");
+ hotelsItem.classList.add("guests-Homes");
+ hotelsItem.innerHTML = `
+   <img class="img_hotel" src="${item.imageUrl}" alt="${item.name}">
+   <div class="text_hotel">${item.name}</div>
+   <div class="location">${item.city}, ${item.country}</div>`;
+ homesInfoPanel.append(hotelsItem);
+});
+$(document).ready(function () {
+  $(".slick__wrapper").slick({
+    slidesToShow: 4,
+    slidesToScroll: 4,
+    infinite: true,
+    arrows: true,
+  });
+});
+ }
+getResponse();
+
+
+
+
+
+
+// слайдер для Avalable
+function slickForAvalableHotels(){
+  $(".available-hotels-items").slick({
+    dots: false,
+    slidesToShow: 4,
+    slidesToScroll: 1,
+    waitForAnimate: false,
+    responsive: [
+      {
+        breakpoint: 1367,
+        settings: {
+          slidesToShow: 5,
+        },
+      },
+      {
+        breakpoint: 1025,
+        settings: {
+          slidesToShow: 4,
+        },
+      },
+      {
+        breakpoint: 769,
+        settings: {
+          slidesToShow: 3,
+        },
+      },
+      {
+        breakpoint: 430,
+        settings: {
+          slidesToShow: 2,
+        },
+      },
+    ],
+  });
+};
+
+const availableHotelsItemsSlider = document.querySelector('.available-hotels-items');
+
+//DOM-elements creatng using fetch data for available Hotels section
+const availableHotelsItemsFromArray = ((data) => {
+  avalableHotels.classList.remove('hide');
```

Comment:
where do you initialize?
![image](https://user-images.githubusercontent.com/20189313/142779572-7fb5a98c-4c4a-4e0e-b35d-464dd04538f0.png)